src/modules/variable/set.rs

```diff
@@ -36,6 +36,10 @@ impl SyntaxModule<ParserMetadata> for VariableSet {
         let variable = handle_variable_reference(meta, tok.clone(), &self.name)?;
         self.global_id = variable.global_id;
         self.is_ref = variable.is_ref;
+        // Check for constant reassignment
+        if variable.is_const {
+            return error!(meta, tok, format!("Cannot reassign constant"))
```

Comment:
Can this error message say a bit more about what variable cannot be reassigned?

Reply:
isn't it obvious?

![image](https://github.com/user-attachments/assets/3174d7c8-3ead-4398-a3c4-768ccc053962)